index.html

```diff
@@ -1,199 +1,326 @@
+        <li><a href="#">Contact</a></li>
+      </ul>
     </header>
     <main>
-        <section class="introduction">
-            <p class="first-intro">I’m Isabella<br>
-                Glad to see you!</p>
-            <p class="second-intro">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <p class="social">LET'S CONNECT</p>
-            <div class="social-icons">
-                <ul class="social-icons">
-                    <li><a href="#"><img src="./images/twitter.svg" alt="twitter icon"></a></li>
-                    <li><a href="https://github.com/Bellagirl-maker/"><img src="./images/github.png" alt="github icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/isabella-otoo-935901146/"><img src="./images/Linkedin icon.svg" alt="Linkedin icon"></a></li>
-                    <li><a href="#"><img src="./images/mail.svg" alt="email"></a></li>
-                    <li><a href="#"><img src="./images/Vector.svg" alt="Angellist"></a></li>
-                </ul>
-            </div> 
-        </section>
-    <section class="projects-section">
+      <section class="introduction">
+        <h1 class="first-intro">
+          I’m Isabella<br />
+          Glad to see you!
+        </h1>
+        <p class="second-intro">
+          I’m a software developer! I can help you build a product , feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="social">LET'S CONNECT</p>
+        <div class="social-icons">
+          <ul class="social-icons">
+            <li>
+              <a href="#"
+                ><img src="./images/twitter.svg" alt="twitter icon"
+              /></a>
+            </li>
+            <li>
+              <a href="https://github.com/Bellagirl-maker/"
+                ><img src="./images/github.png" alt="github icon"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/isabella-otoo-935901146/"
+                ><img src="./images/Linkedin icon.svg" alt="Linkedin icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/mail.svg" alt="email" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/Vector.svg" alt="Angellist" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="projects-section">
         <div class="card-works">
-            <img class="portfolio-card1" src="./images/snapshot1.png" alt="projects screenshot">
-            <img class="portfolio-desktop1" src="./images/Snapshoot Portfolio desktop1.png" alt="project screenshot"> 
```

Comment:
The image class portfolio-card1 and portfolio-desktop1 had the same alt text "projects screenshoot